Cilsil/Program.cs

```diff
@@ -174,7 +174,7 @@ private static IEnumerable<string> GetAssemblies(IEnumerable<string> paths)
                 }
                 catch (FileNotFoundException e)
                 {
-                    Log.WriteError($"Input DLL {e.FileName} not found.");
```

Comment:
Changing these to WriteLine so that they are always written for the user, even when not in debug mode